tests/src/unit/react/formatConversion/export/__snapshots__/html/contextParagraph/basic.html

```diff
@@ -1 +1 @@
-<div data-editable="">React Context Paragraph</div>
\ No newline at end of file
+<div data-editable="" style="white-space: normal;">React Context Paragraph</div>
```

Comment:
why is the white-space added?

Reply:
This is something that's added automatically to the React node view wrapper. Do we need to get rid of it?

Reply:
Ah, yeah - since we render some props out to inline styles for external HTML, I changed it so inline styles are also preserved (previously only attributes were). The React node view wrapper adds that white space style automatically, so it also gets preserved.